docs/integrations/sources/datadog.md

```diff
@@ -21,7 +21,11 @@ An API key is required as well as an API application key. See the [Datadog API a
 7. Enter your `limit` - Number of records to collect per request.
 8. Enter your `start_date` - Optional. Start date to filter records when collecting data from Logs and AuditLogs stream.
 9. Enter your `end_date` - Optional. End date to filter records when collecting data from Logs and AuditLogs stream.
-10. Click **Set up source**.
+10. Enter your `queries` - Optional. Multiple queries resulting in multiple streams.
+    1. Enter the `name`- Required. Query Name.
+    2. Select the `data_source` from dropdown - Required. Supported data sources - metrics, cloud_cost, logs, rum.
+    3. Enter the `query`- Required. A classic query string.
```

Comment:
Please add some examples of `query` and remove the Required its confusing because this field is optional.